polling_stations/apps/file_uploads/templates/file_uploads/uploads_header.html

```diff
@@ -0,0 +1,3 @@
+<a href="{% url 'file_uploads:councils_list' %}">Home</a> •
+<a href="{% url 'file_uploads:file_upload_index' %}">Upload</a> •
+<a href="{% url 'logout' %}">Logout</a>
```

Comment:
TODO: This is an absolutely BARE minimum viable menu.

Style-wise, lets improve it pending https://github.com/DemocracyClub/dc_base_theme/issues/50 it'll do the job

Content-wise, it would be nice to have proper breadcrumbs here